src/hotspot/share/runtime/os.cpp

```diff
@@ -1286,6 +1289,8 @@ void os::print_location(outputStream* st, intptr_t x, bool verbose) {
   }
 
   st->print_cr(INTPTR_FORMAT " is an unknown value", p2i(addr));
+
```

Comment:
Would it make sense to move this a few lines up and print 'unknown' instead of nothing?

Reply:
Right you are.